doc/man1/s_client.pod

```diff
@@ -245,6 +284,16 @@ also used when building the client certificate chain.
 A file containing trusted certificates to use during server authentication
 and to use when attempting to build the client certificate chain.
 
+=item B<-chainCApath directory>
+
+The directory to use for building the chain provided to the server. This
+directory must be in "hash format", see L<verify> for more information.
```

Comment:
Sorry, my mistake this should probably actually be `L<verify(1)>` (it's in man section 1)